android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -65,9 +65,9 @@ class Screen(context: ReactContext?) : FabricEnabledViewGroup(context) {
             val width = r - l
             val height = b - t
 
-            calculateHeaderHeight()
+            val headerHeight = calculateHeaderHeight()
```

Comment:
This breaks on the js stack, it returns the height of the header that doesn't exist